src/App.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+  <div :class="$style.page">
+    <Title />
+    <div :class="$style.body">
+      <Input />
+      <Tasks />
+      <Footer />
+    </div>
+    <div :class="[$style.mask, $style.maskFirst]"></div>
+    <div :class="[$style.mask, $style.maskSecond]"></div>
+  </div>
+</template>
+
+<script>
+import Footer from "@/components/Footer";
+import Title from "@/components/Title";
+import Tasks from "@/components/Tasks";
+import Input from "@/components/Input";
+export default {
+  components: {
+    Footer,
+    Title,
+    Tasks,
+    Input,
+  },
+};
+</script>
+
+<style lang="scss" module>
+* {
+  padding: 0;
+  margin: 0;
+  border: 0;
+   &:before,
+	&:after {
+		box-sizing: border-box;
+	}
+	input {
+		font-family: inherit;
+	}
+	ul li {
+		list-style: none;
+	}
+}
+
+body {
+  background-color: #e0e0e0;
```

Comment:
create scss vars in separate file 
assets/styles/colors.scss and import in vue.config file 
https://vueschool.io/articles/vuejs-tutorials/globally-load-sass-into-your-vue-js-applications/

see also fonts and sizes which you reuse in all project